src/index.ts

```diff
@@ -1,6 +1,34 @@
+/* say
+  Return: (string)
+    '0' if the input is 0
+    'Fizz' if the input is a multiple of 3
+    'Buzz' if the input is a multiple of 5
+    'Fizzbuzz' if the input is a multiple of 3 and 5
+    else the input as a string
+*/
+
+export const say = (input: number) => {
+
+  if (input == 0) {
+    return '0';
+  }
+
+  if (input % 3 == 0 && input % 5 == 0) {
+    return 'Fizzbuzz';
+  }
+
+  if (input % 3 == 0) {
+    return 'Fizz';
+  }
+
+  if (input % 5 == 0) {
+    return 'Buzz';
+  }
+
+  return input.toString();
```

Comment:
Imagine an evolution that requires to handle multiple of 7 ? Could you think of a way to dissociate individual rules from their execution ? What about rule composition ?

Reply:
What do you think about creating a ``rules`` modules which export each individual rules, that are then checked by the ``say`` function and the first rule returning true for the given number will be executed to return the correct text ("Fizz", "Buzz", "Fizzbuzz" or the ``input`` as a string)?

I will implement this idea tomorrow morning.

Reply:
This is exactly what I was thinking about when I mentioned rule composition ;) 
This would also simplifies unit testing as each rule can be tested individually

Reply:
I'm currently working on it 😄